README.rst

```diff
@@ -185,17 +185,19 @@ requirements that are not enabled in those boxes.
 Documentation
 =============
 
-* `Architecture and Roadmap`_
-* `Building the Vagrant box`_ for Developers
 * `Installing the ADB`_
 * `How to use the ADB`_
+** `Using Cockpit with the ADB`_
```

Comment:
@bexelbie  It broke the formatting https://github.com/projectatomic/adb-atomic-developer-bundle#documentation